README.md

```diff
@@ -69,6 +69,67 @@ Now you are ready to run executions, tasks and pipelines.
 
 [app]: https://app.valohai.com
 
+## Running on Valohai with Terminal
+
+### Configure the repository:
+
+To run your code on Valohai using the terminal, follow these steps:
+
+1. Install Valohai on your machine by running the following command:
+```python
```

Comment:
I see that python syntax highlighting has been used here previously, but the code blocks should have the correct language or no language at all. The following blocks are not in python, so we shouldn't use that syntax here. I think **bash** or **shell** would be closer.